lib/tasks/riiif.rake

```diff
@@ -0,0 +1,20 @@
+namespace :riiif do
+  desc 'Clear the riiif cache if it exceeds 50GB'
+  task clear_cache: :environment do
+    # Get the size of the tmp/network_files directory
+    size = `du -sh tmp/network_files`.split("\t").first
+
+    # If the size is over 50GB, delete the oldest files until under 50GB
+    deleted_file_count = 0
+    while !size.nil? && size.match(/^(\d+(?:\.\d+)?)G$/) && Regexp.last_match(1).to_f > 50
```

Comment:
if a file fails to delete, will this become an infinite loop? maybe thats not a possibility

Reply:
Yes, I'm assuming too many things here. I'll beef this up with more error handling tomorrow (e.g. if file fails to delete, if not a file, etc). Thank you!!

Reply:
Okay! Added some error handling so that if removing a file fails, we just log an error and stop. Either we try again and succeed the next day because whatever caused the error was temporary. Or we try again, and fail again, and the directory just keeps growing until the app breaks like it's been doing all along, and we fix it then.